craft_parts/lifecycle_manager.py

```diff
@@ -144,7 +144,7 @@ def clean(self, step: Step = Step.PULL, *, part_names: List[str] = None) -> None
         """
         self._executor.clean(initial_step=step, part_names=part_names)
 
-    def refresh_packages_list(self) -> None:
+    def refresh_packages_list(self) -> None:  # pylint: disable=no-self-use
```

Comment:
you can convert this into a @staticmethod

Reply:
I left self in the interface because other package managers may need it in the future.